pkg/discover/apis.go

```diff
@@ -0,0 +1,51 @@
+package discover
```

Comment:
Nice! In a future patch I'll merge here the Openshift/Vanilla kubernetes

Reply:
yeah I wasn't sure if your `permissions` code could be merged or if there was some additional logic